ch3/docker-compose.yaml

```
@@ -0,0 +1,61 @@
+version: "3.7"
+services:
+  backend-store:
+    image: postgres:14.0
+    container_name: backend-store
+    environment:
+      POSTGRES_USER: myuser
+      POSTGRES_PASSWORD: mypassword
+      POSTGRES_DB: mydatabase
```

```suggestion
      POSTGRES_USER: mlflowuser
      POSTGRES_PASSWORD: mlflowpassword
      POSTGRES_DB: mlflow
```
처럼 mlflow db 임을 명시적으로 적으면 어떨까요

---

좋습니다! 반영하겠습니다 ~!

---

반영되었습니다 :)